packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx

```diff
@@ -115,20 +117,22 @@ function valueFirstDropdown(selectedTagKey: string, trace: Trace) {
   const allSpans = trace.spans;
   // all possibilities that can be displayed
   if (selectedTagKey === serviceName) {
-    const temp = _.chain(allSpans)
-      .groupBy(x => x.process.serviceName)
-      .map((value, key) => ({ key }))
-      .uniq()
-      .value();
+    const temp = _uniq(
+      _map(
+        _groupBy(allSpans, x => x.process.serviceName),
+        (value, key) => ({ key })
+      )
+    );
```

Comment:
If I understand this, the goal is to get a unique list of service names, right? If so, why use groupBy at all, wouldn't this work (in scala syntax): 

spans.map(_.process.serviceName).uniq()

?

Reply:
Yes. I have used a more concise function `uniqBy` here now.